lib/suse-cloud-upgrade-4-to-5-pre

```diff
@@ -38,6 +39,37 @@ fi
 echo_summary "Performing sanity checks..."
 
 
+### Check if repositories are prepared for the upgrade
+# We need to ensure that Cloud5 main repository and update repos are in place
+
+# Skip the repository checks if they are defined in provisioner.json:
+# there might be non-default (and/or network) path
+if [ -f /etc/crowbar/provisioner.json ]; then
+  PROVISIONER_JSON=/etc/crowbar/provisioner.json
+  for repo in SLE-Cloud SUSE-Cloud-5-Pool SUSE-Cloud-5-Updates; do
```

Comment:
In `pre`, the Cloud 5 repos won't be there yet. So I think we should rely on the fact that Cloud 4 repos are listed there to assume that Cloud 5 repos will be listed there.

Hrm. Or we move this check to `post`?